app/helpers/view_components/published_solution.rb

```diff
@@ -0,0 +1,44 @@
+module ViewComponents
+  class PublishedSolution < ViewComponent
+    extend Mandate::Memoize
+
+    def initialize(solution)
+      super()
+
+      @solution = solution
+    end
+
+    def to_s
+      ApplicationController.render "components/published_solution",
+        locals: {
+          exercise: solution.exercise,
+          exercise_title: solution.exercise.title,
+          track_title: solution.track.title,
+          published_at: solution.published_at,
+          stars_count: 10, # TODO
```

Comment:
We're using `num_` elsewhere as the prefix for a count. Do we want to use that here too?

Reply:
Good catch. Thanks 🙂